widget.subscribe.php

```diff
@@ -353,7 +353,7 @@ function add_subscriber() {
 							}
 						} else {
 							$var_name          = str_replace( 'custom_', '', $name_stripped );
-							$vars[ $var_name ] = sanitize_text_field( $_POST[ 'custom_' . $name_stripped ] );
+                                                        $vars[$var_name] = stripslashes(sanitize_text_field($_POST['custom_' . $name_stripped]));
```

Comment:
lets also follow spacing convention as per the current line for e.g. [ $var_name ] vs [$var_name]

Reply:
Changes made